make/Makefile.container.mk

```diff
@@ -62,3 +62,47 @@ endif
 
 ## container-push: Pushes all container images to quay
 container-push: container-push-kiali-quay
+
+# Ensure "docker buildx" is available and enabled. For more details, see: https://github.com/docker/buildx/blob/master/README.md
+.ensure-docker-buildx:
```

Comment:
```suggestion
.ensure-docker-buildx:
	@required_docker="19.03.13"; \
	current_docker="$$(docker version --format '{{.Client.Version}}')"; \
	if [ "$$(printf $$required_docker\\n$$current_docker | sort -V | head -n1)" != "$$required_docker" ]; then \
	  echo "You have an older 'docker' [$$current_docker] which is not able to perform this build. Please upgrade to at least [$$required_docker]"; \
	  exit 1; \
	else \
	  echo "Docker is version [$$current_docker] which is the same or newer than the required [$$required_docker] - build can continue"; \
	fi
```

Reply:
@jshaughn 's issue not for docker. I'm not sure which OS he used. may MacOS ?

docker desktop includes a buildx. but not includes in docker for Linux distro

```
$ docker buildx version
github.com/docker/buildx v0.4.2 fb7b670b764764dc4716df3eba07ffdae4cc47b2
```

so here just need to check the buildx version

Reply:
Ahh.. ok, so we should be looking at the buildx version, not the docker version. That makes sense because his error was his buildx did not accept the `--builder` option. So we just have to tweek my suggested fix to look at the buildx version and not the docker version.

Reply:
Yeah. I'm trying to do this.